pkgs/applications/science/biology/svaba/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, zlib, bzip2, lzma, fetchFromGitHub } :
+
+stdenv.mkDerivation rec {
+  version = "1.1.0";
+  pname = "svaba";
+
+  src = fetchFromGitHub {
+    owner = "walaj";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1vv5mc9z5d22kgdy7mm27ya5aahnqgkcrskdr2405058ikk9g8kp";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [ zlib bzip2 lzma ];
```

for autotool builds, use autoreconfHook, this will set the correct install paths for the build.
```suggestion
  nativeBuildInputs = [ autoreconfHook ];

  buildInputs = [ zlib bzip2 lzma ];
```

---

disregard, autoconfHook, doesn't add anything in this case.